packages/crypto/src/index.ts

```diff
@@ -1 +1,9 @@
+export { ctr } from "@noble/ciphers/aes";
+export { hmac } from "@noble/hashes/hmac";
+export { sha256 } from "@noble/hashes/sha256";
+export { sha512 } from "@noble/hashes/sha512";
+export { keccak_256 } from "@noble/hashes/sha3";
```

Comment:
```suggestion
export { keccak_256 as keccak256 } from "@noble/hashes/sha3";
```

we should avoid mixing snake_case with camelCase in a project if possible

Reply:
Done.